buckifier/targets_cfg.py

```diff
@@ -19,6 +19,7 @@
   "-DROCKSDB_SCHED_GETCPU_PRESENT",
   "-DROCKSDB_SUPPORT_THREAD_LOCAL",
   "-DOS_LINUX",
+  "-DROCKSDB_UBSAN_RUN",
```

Comment:
Did this flag get in here by mistake?

Reply:
It does exist in the current version of the TARGET file. I am guessing it was removed from the buckifier by mistake.

Reply:
Oh, ok. Thanks. Since nothing was specified about adding this flag in the title or description, I assumed it got in here by mistake.
I prefer to add a line about this too to the summary.

Reply:
The use of ROCKSDB_UBSAN_RUN is removed in #2776. We should remove it and update TARGETS.